src/main/java/org/ibp/api/java/impl/middleware/common/validator/GermplasmUpdateValidator.java

```diff
@@ -135,14 +144,23 @@ public void validateLocationAbbreviation(final BindingResult errors, final Strin
 
 	}
 
-	public void validateBreedingMethod(final List<GermplasmUpdateDTO> germplasmUpdateDTOList) {
+	public void validateBreedingMethod(final BindingResult errors, final String programUUID,
+		final List<GermplasmUpdateDTO> germplasmUpdateDTOList) {
 
-		final Set<String> breedingMethodsAbbrs =
+		final List<String> breedingMethodsAbbrs =
 			germplasmUpdateDTOList.stream().filter(dto -> StringUtils.isNotEmpty(dto.getBreedingMethodAbbr()))
-				.map(dto -> dto.getBreedingMethodAbbr()).collect(Collectors.toSet());
+				.map(dto -> dto.getBreedingMethodAbbr()).collect(Collectors.toList());
```

Comment:
use `GermplasmUpdateDTO::getBreedingMethodAbbr` instead of `dto -> dto.getBreedingMethodAbbr()`